atam4j/src/main/java/me/atam/atam4j/TestRunListener.java

```diff
@@ -10,10 +10,7 @@
 import org.slf4j.LoggerFactory;
 
 import java.lang.reflect.Method;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Objects;
+import java.util.*;
```

Comment:
Another thought - we should avoid changes to code that are not in the scope of the PR.  :)